src/pages/addFlower/AddFlower.js

```diff
@@ -11,22 +9,18 @@ const AddFlower = () => {
     <div className="addflower-page">
       <header className="head-of-addflower">
         <span>
-          <RiPlantLine />
+          <h3>🌱</h3>
```

Comment:
emoji in not a good thing to be added inside of h3, it is better to have it inside of another element instead of h3